modules/ROOT/nav.adoc

```diff
@@ -1,8 +1,9 @@
 * xref:faq.adoc[FAQ]
 * xref:getting-started.adoc[Getting Started]
-* xref:bare-metal.adoc[Installing on Bare Metal]
 * Provisioning Machines
 ** xref:stream-metadata.adoc[Stream metadata]
+** xref:bare-metal.adoc[Installing on Bare Metal]
+** xref:live-booting-ipxe.adoc[Live-booting via iPXE (transient system)]
```

Comment:
Maybe we could drop `(transient system)` to save pixels, since it's redundant with `Live-booting`.